### PR TITLE
feat(web): send processing window redesign

### PR DESCRIPTION
**Pull request by turisap**

### CODE REVIEW

File: packages/ui/src/styles/themes/light.ts [hunk 1]
```diff
@@ -39,6 +39,12 @@ const theme = {
       buttonDisabled: '#D0D3E9',
     },
   },
+  zIndex: {
```
Comment (beautyfree) [inline packages/ui/src/styles/themes/light.ts:42]: могу уверенно сказать что это должно быть не в теме, т.к врятли эти значения поменяются в темной теме

Comment (turisap) [reply packages/ui/src/styles/themes/light.ts]: вынес в отдельный файл и сделал `css` переменными

File: packages/web/src/utils/analytics/types.d.ts [hunk 2]
```diff
@@ -201,12 +201,12 @@ export type TrackEventType = {
   // (event: 'send_address_keydown'): void;
   // // Пользователь нажимает "Send"
   // (event: 'send_send_click', data: { tokenTicker: string; sum: number }): void;
-  // // Пользователь нажимает "Close"
-  // (event: 'send_close_click', data: { transactionConfirmed: boolean }): void;
+  // Пользователь нажимает "Close"
+  (event: 'send_close_click', data: { transactionConfirmed: boolean }): void;
```
Comment (beautyfree) [inline packages/web/src/utils/analytics/types.d.ts:205]: These are deprecated events. Wrote to PM

File: packages/web/src/components/modals/TransactionInfoModals/TransactionSwap/TransactionSwap.tsx [hunk 3]
```diff
@@ -0,0 +1,370 @@
...
+  }
+};
+
+const DEFAULT_TRANSACTION_ERROR = 'Transaction error';
+
+type SendActionType = () => Promise<string>;
+type SwapActionType = () => Promise<string>;
+
+export type TransactionStatusModalProps = {
+  type: 'send' | 'swap';
```
Comment (beautyfree) [inline packages/web/src/components/modals/TransactionInfoModals/TransactionSwap/TransactionSwap.tsx:108]: I think you know that if it modal for Swap so it should have only one type and conditions in code must be removed. 
It's just notice

File: packages/web/src/components/modals/TransactionConfirmModal/TransactionConfirmModal.tsx [hunk 4]
```diff
@@ -28,6 +28,11 @@ import { Swap } from './Swap';
 
 const WrapperModal = styled(Modal)`
   flex-basis: 524px;
+  max-height: 80vh;
+
+  &:focus-visible {
```
Comment (beautyfree) [inline packages/web/src/components/modals/TransactionConfirmModal/TransactionConfirmModal.tsx:33]: I think that this case with focus can be reproduced on other modals too

File: packages/web/src/app/contexts/general/modals/modals.tsx [hunk 5]
```diff
@@ -80,6 +80,10 @@ const modalsMap = new Map<ModalType, LoadableComponent<ModalPropsType & any>>([
     ModalType.SHOW_MODAL_TRANSACTION_STATUS,
     loadable(() => import('components/modals/TransactionInfoModals/TransactionStatusModal')),
   ],
+  [
+    ModalType.SHOW_MODAL_TRANSACTION_STATUS_SWAP,
```
Comment (beautyfree) [inline packages/web/src/app/contexts/general/modals/modals.tsx:84]: You should know that constants have names like files, so this constant should be SHOW_MODAL_TRANSACTION_WAP if file called TransactionSwap

File: packages/ui/src/styles/helpers/zIndex.ts [hunk 6]
```diff
@@ -0,0 +1,12 @@
+import { generate } from '../themes/utils';
+
+const vars = {
```
Comment (beautyfree) [inline packages/ui/src/styles/helpers/zIndex.ts:3]: It's not reasonable practice if this constants will not be changing dynamically. You should write them in constants way like in other helpers.

**Review by beautyfree — APPROVED** [review]

**Review by agroznykh — APPROVED** [review]

**Pull request merged**